site/docs/pt/guide/filter-queries.md

```diff
@@ -0,0 +1,355 @@
+
+```ts
+// Posts de canal enviados por `ctx.senderChat`
+bot.on("channel_post");
+// Encaminhamento automático do canal `ctx.senderChat`:
+bot.on("message:is_automatic_forward");
+// Mensagens normais enviadas por `ctx.from`
+bot.on("message").filter((ctx) => ctx.senderChat === undefined);
+// Admin anônimo em `ctx.chat`
+bot.on("message").filter((ctx) => ctx.senderChat?.id === ctx.chat.id);
+// Usuários enviando mensagens em nome de seu canal `ctx.senderChat`
+bot.on("message").filter((ctx) =>
+  ctx.senderChat !== undefined && ctx.senderChat.id !== ctx.chat.id
+);
+```
+
+### Filtrando por Propriedades do Usuário
+
+Se você quiser filtrar por outras propriedades de um usuário, precisará realizar uma request adicional, por exemplo, `await ctx.getAuthor()` para o autor da mensagem.
+As filter queries não realizarão requests adicionais para você.
+Ainda é simples realizar esse tipo de filtragem:
+
+```ts
+bot.on("message").filter(
+  async (ctx) => {
+    const user = await ctx.getAuthor();
+    return user.status === "creator" || user.status === "administrator";
+  },
+  (ctx) => {
+    // Trata mensagens de criadores e administradores.
+  },
+);
+```
+
+### Reutilizando a Lógica de Uma Filter Query
+
+Internamente, o `bot.on` depende de uma função chamada `matchFilter`.
+Ela recebe uma filter query e a compila para uma função de predicado.
+O predicado é simplesmente passado para `bot.filter` para filtrar atualizações.
+
+Você pode importar `matchFilter` diretamente se quiser usá-lo em sua própria lógica.
+Por exemplo, você pode decidir descartar todas as atualizações que correspondem a uma determinada query:
+
+```ts
+// Descarta todas as mensagens de texto ou posts de text em canais.
+bot.drop(matchFilter(":text"));
+```
+
+Analogamente, você pode fazer uso dos tipos de filter query que o grammY usa internamente:
+
+### Reutilizando os Tipos de Filter Query
+
+Internamente, `matchFilter` usa os [type predicates](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates) do TypeScript para ajustar o tipo de `ctx`.
+Ela recebe um tipo `C extends Context` e um `Q extends FilterQuery` e produz `ctx is Filter<C, Q>`.
+Em outras palavras, o tipo `Filter` é o que você realmente recebe como seu `ctx` no middleware.
+
+Você pode importar `Filter` diretamente se quiser usá-lo em sua própria lógica.
+Por exemplo, você pode decidir definir uma função de handler que trata objetos de contexto específicos que foram filtrados por uma filter query:
+
+```ts
+function handler(ctx: Filter<Context, ":text">) {
+  // trata objeto de contexto ajustado
+}
+
+bot.on(":text", handler);
+```
+
+> Confira as referências da API para [`matchFilter`](https://deno.land/x/grammy/filter.ts?s=matchFilter), [`Filter`](https://deno.land/x/grammy/filter.ts?s=Filter) e [`FilterQuery`](https://deno.land/x/grammy/filter.ts?s=FilterQuery) para ler mais.
```

Comment:
```suggestion
> Confira as referências de API para [`matchFilter`](https://deno.land/x/grammy/filter.ts?s=matchFilter), [`Filter`](https://deno.land/x/grammy/filter.ts?s=Filter) e [`FilterQuery`](https://deno.land/x/grammy/filter.ts?s=FilterQuery) para ler mais.
```

I think we are using API Reference always as "Referência de API". At least we accorded that